hardware/asic/README.md

```diff
@@ -242,6 +242,7 @@ git clone https://github.com/VLSIDA/OpenRAM
   ```bash
   export OPENRAM_HOME="$HOME/openram/compiler"
```

Comment:
mind the suggested name changes 

you can't assume openram is installed in $HOME


Instead ask the user to 

export OPENRAM_HOME=path/to/openram

so this can be any directory the user may like

Reply:
Ok I got it sir I am going to change it now